website/docs/docs/deploy/single-tenant.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: single-tenant-deployment
-title: Single Tenant
+title: Single tenant 
```

Comment:
I heard we're rebranding single tenant as virtual private cloud (VPC).  Should we address that change here or better to wait until after IA is live and formal announcements made?

Reply:
good callout! 

yeah, waiting sounds good to me. if the product team reaches out to us, we can always open a PR async to address it too